rust/src/lib/net_state.rs

```diff
@@ -121,6 +121,26 @@ pub struct NetworkState {
     pub(crate) memory_only: bool,
 }
 
+impl std::fmt::Debug for NetworkState {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        let mut debug_struct = f.debug_struct("NetworkState");
+        debug_struct.field("hostname", &self.hostname);
+        debug_struct.field("dns", &self.dns);
+        debug_struct.field("rules", &self.rules);
+        debug_struct.field("routes", &self.routes);
+        debug_struct.field("interfaces", &self.interfaces);
+        debug_struct.field("ovsdb", &self.ovsdb);
+        debug_struct.field("timeout", &"<_password_hid_by_nmstate>");
+        debug_struct.field("no_verify", &"<_password_hid_by_nmstate>");
+        debug_struct.field("no_commit", &"<_password_hid_by_nmstate>");
```

Comment:
Why these fields are filled with value `_password_hid_by_nmstate`  ?

Reply:
I intended to hide password and other secrets, cause from the code so far and info, so i used them as place holders during the debugging process or logs

Reply:
`no_verify` and other properties are not secrets.